workflows/cerbos/policies/workflow_run.yaml

```diff
@@ -22,6 +22,12 @@ resourcePolicy:
       effect: EFFECT_ALLOW
       derivedRoles:
         - project_member
+    
+    - actions: ['create', 'update', 'view', 'delete', 'download']
+      effect: EFFECT_ALLOW
+      derivedRoles:
+        - single_user_owner
+        
```

Comment:
1. Can you just add `single_user_owner` to the list of derived_roles for the other rules, instead of creating a new rule?
2. Do we also need this permissions change for workflowRunEntityInputs?

Reply:
1.  Sure, makes sense! 
2. I'm not sure how the `workflowRunEntityInputs` are being used. I suppose they won't show up if we query for them...

Reply:
Maybe check with Suzette, I'm not sure if she needs to access `workflowRunEntityInputs` to get some display data, or if she can get everything she needs from the `workflowRun`'s raw inputs + the `bulkDownload` entity.

Reply:
I was assuming (?) that entities are inputs to the bulk download workflow run. If that's not the case, then we probably don't need to change it.